flox/core.py

```diff
@@ -589,28 +609,39 @@ def chunk_reduce(
     dict
     """
 
-    if dtype is not None:
-        assert isinstance(dtype, Sequence)
-    if fill_value is not None:
-        assert isinstance(fill_value, Sequence)
-
-    if isinstance(func, str) or callable(func):
-        func = (func,)  # type: ignore
+    if not (isinstance(func, str) or callable(func)):
```

Comment:
can we use `_atleast_1d` here? It's in `aggregations.py` but we could move it to a `utils.py`

Reply:
Can we do this in a different PR? 
I think typing atleast_1d correctly will not be trivial and I don't have enough motivation right now to continue quarreling with mypy.

Reply:
see #156